apps/docs/package.json

```diff
@@ -11,7 +11,9 @@
         "clear": "docusaurus clear",
         "serve": "docusaurus serve",
         "write-translations": "docusaurus write-translations",
-        "write-heading-ids": "docusaurus write-heading-ids"
+        "write-heading-ids": "docusaurus write-heading-ids",
+        "format": "remark ./**/*.mdx --frail",
```

Comment:
```suggestion
        "format": "remark ./**/*.mdx --frail --quiet",
```

Reply:
What about adding `--quiet` so that it only shows warnings and errors?

https://github.com/remarkjs/remark/tree/main/packages/remark-cli#cli